crates/red_knot_python_semantic/resources/mdtest/annotations/unsupported_special_forms.md

```diff
@@ -18,7 +18,7 @@ def f(*args: Unpack[Ts]) -> tuple[Unpack[Ts]]:
     # TODO: should understand the annotation
     reveal_type(args)  # revealed: tuple
 
-    reveal_type(Alias)  # revealed: @Todo(Unsupported or invalid type in a type expression)
+    reveal_type(Alias)  # revealed: @Todo(Invalid or unsupported `KnownInstanceType` in `Type::to_type_expression()`)
```

Comment:
These assertions are failing in the release build, I think because `TODO_METADATA_REGEX` in the mdtests matcher can't handle a close parenthesis in the Todo message. Might not be easy to fix that in mdtest (a greedy match could fail if there are multiple Todo types in a union). Simplest fix is to remove the parens from this todo message.